docs/Documentation/Configuration/Commands-and-permissions.md

```diff
@@ -22,6 +22,8 @@ Optional arguments are displayed as such: `[argument]`.
 * `/q event <playerName> <package.eventID>` - Fires an event for the player
 * `/q condition <playerName> <package.conditionID>` - Shows if the player meet specified condition or not
 * `/q item <package.itemID>` - Creates an item based on what you're holding in hand
+* `/q variable <playerName> <package.objectiveID> <list/set> [key/filter] [value]` - Lists or sets online player's 
+active variable objective's values, similar to the [variable event](../Scripting/Building-Blocks/Events-List.md#variable-variable)
```

Comment:
Is the addition `- To remove a key/value pair, the value after the key must be omitted` a good formulation?

Reply:
The whole sentence as it is at the moment is already too complex.
Based on the surrounding examples I would simply write something like:
> Manage variables stored in `variable`(+link) objectives

And add a detailed description + explanation further down in the document.